packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -145,7 +147,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     let ariaDescribedBy;
     if (ariaDescription) {
       ariaDescribedBy = _ariaDescriptionId;
-    } else if (description) {
+    } else if (secondaryText || description) {
```

Comment:
you can also do this:

```
const {
  description,
  secondaryText = description
} = this.props;
```

Reply:
Thanks, I just first noticed this in another PR yesterday. Fixing.